charts/apisix-dashboard/README.md

```diff
@@ -73,6 +73,8 @@ _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documen
 | config.conf.log.accessLog.filePath | string | `"/dev/stdout"` | Error log path |
 | config.conf.log.errorLog | object | `{"filePath":"/dev/stderr","level":"warn"}` | Error log level. Supports levels, lower to higher: debug, info, warn, error, panic, fatal |
 | config.conf.log.errorLog.filePath | string | `"/dev/stderr"` | Access log path |
+| config.conf.plugins | list | `[]` |  |
```

Comment:
Need to add description of the properties

Reply:
Readme file is auto-generated by `make helm-docs`. The field comment is using format like this:

```yaml
# -- comment goes here
fieldName: value
```

Reply:
Okay awesome 🙏